indico/modules/rb/notifications/reservations.py

```diff
@@ -104,9 +107,10 @@ def notify_confirmation(reservation, reason=None):
 @email_sender
 def notify_creation(reservation):
     notification = ReservationNotification(reservation)
+    prebooking = not reservation.is_accepted
```

Comment:
This seems equivalent:
```suggestion
    prebooking = reservation.is_pending
```

Reply:
indeed, i had copied it from the template :D